subsys/bluetooth/shell/gatt.c

```diff
@@ -1096,6 +1106,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(gatt_cmds,
 	SHELL_CMD_ARG(set, NULL, "<handle> [data...]", cmd_set, 2, 255),
 	SHELL_CMD_ARG(show-db, NULL, "[uuid] [num_matches]", cmd_show_db, 1, 2),
 #if defined(CONFIG_BT_GATT_DYNAMIC_DB)
+	SHELL_CMD_ARG(att_mtu, NULL, "Output ATT MTU size", cmd_att_mtu, 1, 0),
```

Comment:
Should be moved out of ifdef' DYNAMYC_DB.

Reply:
Yes.  I intended to do that, must have missed.  Will fix.